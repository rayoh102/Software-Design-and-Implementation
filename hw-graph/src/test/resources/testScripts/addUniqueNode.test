# Tests multiple node addition using unique data

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add another node
AddNode graph1 n2

# See if nodes are in graph
ListNodes graph1