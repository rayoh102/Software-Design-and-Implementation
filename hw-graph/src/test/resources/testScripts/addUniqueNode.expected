# Tests multiple node addition using unique data

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add another node
added node n2 to graph1

# See if nodes are in graph
graph1 contains: n1 n2